Software/Development/SemesterProject/System/systemInit.c

```diff
@@ -148,9 +150,12 @@ void systemInit(void)
     initPWM();
     initAnalogSignals();      // Initialize the analog signals and their ADC channels
     initializeGUIPushbuttonsStructure();
-	  initDigitalOutputs();
-	  initPushbuttons();
+    initDigitalOutputs();
+    initPushbuttons();
     initWatchdog();
+    UARTinit();
+    UARTStringPrint("\n\rWelcome to the excellent Go-Kart controller, by PED-841.\n\r");
```

Comment:
why not including this in UARTinit(), or even better in initUART() ¿?

Reply:
Done